apps/pie-storybook/stories/pie-textarea.stories.ts

```diff
@@ -42,6 +93,48 @@ const textareaStoryMeta: TextareaStoryMeta = {
                 summary: defaultProps.resize,
             },
         },
+        name: {
+            description: 'The name of the textarea (used as a key/value pair with `value`). This is required in order to work properly with forms.',
+            control: 'text',
+            defaultValue: {
+                summary: defaultArgs.name,
+            },
+        },
+        value: {
+            description: 'The value of the textarea (used as a key/value pair in HTML forms with `name`).',
```

Comment:
just triple checking, is this correct for textarea values in forms - it's still a key/value pair with `name`?

Reply:
When the form is submitted, the content of the <textarea> is sent as a key/value pair where the key is the value of the name attribute, and the value is the content entered by the user.